icu4j/.gitignore

```diff
@@ -0,0 +1 @@
+/release_cldr
```

Comment:
this build creates this dir, i think it's an oversight that this wasn't included..